Compiler error on Linux

When installing with `cargo install what` with Rust 1.39.0.

```rust
error[E0107]: wrong number of type arguments: expected 2, found 0
  --> /home/mmstick/.cargo/registry/src/github.com-1ecc6299db9ec823/what-0.4.0/src/network/dns/resolver.rs:10:21
   |
10 | pub struct Resolver(AsyncResolver);
   |                     ^^^^^^^^^^^^^ expected 2 type arguments

error: aborting due to previous error

For more information about this error, try `rustc --explain E0107`.
error: failed to compile `what v0.4.0`, intermediate artifacts can be found at `/tmp/cargo-install5fzOgK`

Caused by:
  could not compile `what`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -2,7 +2,7 @@
 edition = "2018"
 name = "what"
 description = "Display current network utilization by process, connection and remote IP/hostname"
-version = "0.4.0"
+version = "0.4.1"
 homepage = "https://github.com/imsnif/what"
 repository = "https://github.com/imsnif/what"
 readme = "README.md"
@@ -27,8 +27,8 @@ failure = "0.1.6"
 chrono = "0.4"
 regex = "1.3.1"
 lazy_static = "1.4.0"
-tokio = { version = "^0.2", features = ["rt-core", "sync"] }
-trust-dns-resolver = "0.18.0-alpha.2"
+tokio = { version = "0.2", features = ["rt-core", "sync"] }
+trust-dns-resolver = "=0.18.0-alpha.2"
 async-trait = "0.1.21"
 
 [target.'cfg(target_os="linux")'.dependencies]

